src/subcommands/deploy.rs

```diff
@@ -51,6 +51,8 @@ pub struct Deploy {
     )]
     estimate_only: bool,
     #[clap(long, help = "Wait for the transaction to confirm")]
+    salt: Option<String>,
+    #[clap(long, help = "Use the given salt to compute contract deploy address")]
```

Comment:
Did you add the help text on the wrong line?

Reply:
Other good catch... sorry for those mistakes.